library/templates/usdot_airports.yml

```diff
@@ -3,31 +3,30 @@ dataset:
   version: "{{ version }}"
   acl: public-read
   source:
-    url:
-      path: https://opendata.arcgis.com/datasets/831853ab8b714a81b6a3e21d0b164a4e_0.geojson
-      subpath: ""
+    script: *name
```

Comment:
I think the `subpath` should be explicit in the template or mentioned in the description notes. it can get a little confusing as to where the xlsx file should be read in from without the explicit 'subpath' i.e. library/tmp/all-airport-data.xlsx as supposed to the root directory (the python script does read it in from library/tmp/ but might want to have it in the template to match other manual templates)

Reply:
hey @mbh329 about this comment. Could you provide an example where the subpath is provided when a python script is running? I checked a few and wasn't able to find any such example